semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/rpc/RpcSemanticConventionUtils.java

```diff
@@ -268,4 +301,36 @@ public static Boolean check(Map<String, AttributeValue> avm) {
     }
     return false;
   }
+
+  public static Optional<String> getRpcPath(Event event) {
+
+    String service = getRpcService(event).orElse("");
+    String method = getRpcMethod(event).orElse("");
+
+    if (StringUtils.isNotBlank(service) && StringUtils.isNotBlank(method)) {
+      return Optional.of(DOT_JOINER.join(service, method));
+    }
+
+    return Optional.empty();
+  }
+
+  private static Optional<String> getRpcMethod(Event event) {
+    return Optional.ofNullable(
+        SpanAttributeUtils.getFirstAvailableStringAttribute(
+            event, List.of(OTEL_SPAN_TAG_RPC_METHOD.getValue())));
+  }
+
+  /**
+   * Takes in a param path and converts it to a DOT joined path /service.product/GetProducts ->
+   * service.product.GetProducts
+   *
+   * @param path
+   * @return
+   */
+  private @Nullable static String sanitizePath(@Nonnull String path) {
```

Comment:
its the same method defined again, consider adding it in a common class

Reply:
I was also thinking about this but couldn't find the suitable class for shifting common code.